packages/client/src/components/app/blocks/RowExplorer.svelte

```diff
@@ -0,0 +1,247 @@
+<script>
+  import Block from "components/Block.svelte"
+  import BlockComponent from "components/BlockComponent.svelte"
+  import { makePropSafe as safe } from "@budibase/string-templates"
+  import { generate } from "shortid"
+
+  export let dataSource
+  export let height
+
+  export let cardTitle
+  export let cardSubtitle
+  export let cardDescription
+  export let cardImageURL
+  export let cardSearchField
+
+  export let detailFields
+  export let detailTitle
+
+  const stateKey = generate()
+
+  let listDataProviderId
+  let listRepeaterId
+</script>
+
+<Block>
+  <BlockComponent
+    type="container"
+    props={{
+      direction: "row",
+    }}
+    styles={{
+      custom: `
+        height: ${height} !important;
+      `,
+    }}
+  >
+    <BlockComponent
+      type="dataprovider"
+      order={0}
+      bind:id={listDataProviderId}
+      props={{
+        dataSource,
+        paginate: true,
+        limit: 10,
+        filter: [
+          {
+            id: generate(),
+            field: cardSearchField,
+            operator: "fuzzy",
+            type: "string",
+            value: `{{ [state].[${stateKey}-search] }}`,
```

Comment:
While it's usually fine to manually escape HBS expressions by just wrapping them in square brackets, it's better to always use the makePropSafe (imported here as safe) util to be sure.

I think it would be good to update all references of manual escaping to use that util. I'll not add comments for every usage since there are loads - but I think it's best to get them all updates.

It also means we can upate that util in future and know that all our bindings will be correctly escaped still.

Reply:
I wrapped everything including things that weren't variables in safe, not sure if that's correct or not.